core/spec/models/spree/promotion/actions/create_line_items_spec.rb

```diff
@@ -7,6 +7,11 @@
   let(:shirt) { create(:variant) }
   let(:mug) { create(:variant) }
 
+  def empty_stock(variant)
+    variant.stock_items.update_all(backorderable: false)
+    variant.stock_items.each { |i| i.reduce_count_on_hand_to_zero }
```

Comment:
Pass `&:reduce_count_on_hand_to_zero` as an argument to `each` instead of a block.